packages/AlgernonTests.package/ALGTestUI.class/instance/testAlgernonPositioningWindowIsSmallerAfterReopening.st

```diff
@@ -0,0 +1,7 @@
+testing
+testAlgernonPositioningWindowIsSmallerAfterReopening
+	| point |
+	point := 1 @ 1.
+	self algerInstance lastWorldBounds: World bounds corner + point.
+	self algerInstance activate.
+	self assert: ((self algerInstance center x = self algerInstance class defaultPosition x rounded) & (self algerInstance center y = self algerInstance class defaultPosition y rounded))
```

Comment:
You don't move the Algernon Instance in this Test, therefore this will always succeed.

Reply:
Algernon is repositioned through the check between lastWorldBounds and the current world bounds when opened. Therefore it will reposition

Reply:
Algernon is in center of the World at the start of the test; If your methods run it would be positioned in the center of the world. If your methods don't run/do nothing it would still be in the center of the world. Therefore the tests will allways succeed, wether your logic is working or not.
(To verify just comment out the "activate" statement, which runs your logic; the tests will work nevertheless)